frontend/src/app/modules/global_search/global-search-input.component.html

```diff
@@ -8,20 +8,39 @@
         <div class="block-5"></div>
       </div>
     </div>
-    <input #inputEl
-           type="text"
-           name="q"
-           id="q"
-           size="20"
-           class="top-menu-search--input"
-           [class.-expanded]="expanded"
-           placeholder="{{text.search}}"
-           autocomplete="off"
-           [ngModel]="searchTerm"
-           (focus) = "resize()"
-           (focusout) = "resize()"
-           (keyup.enter)="submitNonEmptySearch()"
-           (keydown.escape)="inputEl.blur()">
+     <ng-select [items]="results"
+                name="q"
+                id="q"
+                [ngClass]="'top-menu-search--input'"
+                [class.-expanded]="expanded"
+                placeholder="{{text.search}}"
+                bindLabel="subject"
+                (search)="handleUserInput($event)"
+                (focus)="onFocus()"
+                (focusout)="onFocusOut()"
+                (open)="toggleMenu()"
+                (change)="onChange($event)"
+                [virtualScroll]="true"
+                [searchFn]="customSearchFn">
+        <ng-template ng-option-tmp let-item="item" let-index="index" let-search="searchTerm">
+          <div *ngIf="!item.id; else isWorkPackageItem">
+            <div tabindex="-1" class="ui-menu-item">
+              <div class="ui-menu-item-wrapper">
+                <span class="search-autocomplete--search-term"> {{searchTerm}} </span>
+                <span class="search-autocomplete--project-scope" title="{{item.projectScope}}"> {{item.subject}} ↵ </span>
+              </div>
+            </div>
+          </div>
+          <ng-template #isWorkPackageItem>
```

Comment:
This got me a bit confused as `isWorkPackageItem` sounds like a state and not so much like the name of a template. Would you consider renaming it?